javascript-forms/form-events.js

```diff
@@ -0,0 +1,22 @@
+function handleFocus(event) {
+  console.log('focus event fired');
+  console.log('event.target.name', event.target.name);
+}
+
+function handleBlur(event) {
+  console.log('blur event fired');
+  console.log('event.target.name', event.target.name);
+}
+
+function handleInput(event) {
+  console.log('value of', event.target.name, event.target.value);
+}
+
+var $inputArray = document.querySelectorAll('input');
+var $inputName = $inputArray[0];
+var $inputEmail = $inputArray[1];
+var $textcontent = document.querySelector('textarea');
+
+$inputName.addEventListener('focus', handleFocus);
+$inputEmail.addEventListener('blur', handleBlur);
+$textcontent.addEventListener('input', handleInput);
```

Comment:
![Screen Shot 2021-02-03 at 11 21 06 AM](https://user-images.githubusercontent.com/32880187/106797822-fa046000-6611-11eb-99fe-92e2a0d488bf.png)

You are missing some event handlers here. Each form control should have an event listener for each of these events.